docker-compose.yml

```diff
@@ -7,7 +7,8 @@ services:
     ports:
       - "3306"
     env_file:
-      - ./local.env
+      - path: ./local.env
```

Comment:
with the addition of cert stuff, are the user name, password, etc. in local.env no longer needed? If so, shouldn't the local.env.dist be edited too?